kitsune/wiki/tasks.py

```diff
@@ -239,8 +227,8 @@ def _rebuild_kb_chunk(data):
         if message:
             log.debug(message)
             messages.append(message)
-    d = time.time() - start
-    statsd.timing('wiki.rebuild_chunk', int(round(d * 1000)))
+    elapsed_time = int(round((time.time() - start) * 1000))
+    log.debug(f'wiki rebuild task chuck time: {elapsed_time}')
```

Comment:
We don't need this, celery times our tasks for us, and the durations can be seen quite easily through flower.